packages/bento-design-system/src/List/ListItem.tsx

```diff
@@ -54,29 +55,31 @@ type RightItem = {
   trailingIcon?: (props: IconProps) => JSX.Element;
 };
 
-type Props = Kind &
-  LeftItem &
-  RightItem & {
-    disabled?: boolean;
-    size: ListSize;
-    isFocused?: boolean;
-    isSelected?: boolean;
-    ignoreTabIndex?: boolean;
-    key?: string | number;
-  } & (
-    | {
-        onPress?: () => void;
-        href?: never;
-        target?: never;
-      }
-    | {
-        href?: string;
-        target?: AnchorHTMLAttributes<HTMLAnchorElement>["target"];
-        onPress?: never;
-      }
-  );
+type CommonProps = {
+  disabled?: boolean;
+  size: ListSize;
+  isFocused?: boolean;
+  isSelected?: boolean;
+  ignoreTabIndex?: boolean;
+  key?: string | number;
+  ref?: RefObject<HTMLElement>;
```

Comment:
I don't think this is necessary, `forwardRef` automatically adds `ref` to props.